server.js

```diff
@@ -3,16 +3,20 @@
 require('dotenv').config();
 const express = require('express');
 const cors = require('cors');
-
+const mongoose = require('mongoose');
+const handleGetBooks = require('./modules/getBooks');
 const app = express();
 app.use(cors());
-
 const PORT = process.env.PORT || 3001;
 
-app.get('/test', (request, response) => {
+mongoose.connect(process.env.MONGO_DB_LIVE_URL);
 
-  response.send('test request received')
+const Book = require('./modules/bookModel.js'); // do we need here?
```

Comment:
- line 14 is not needed here, since you are using it in your `seed.js` module. 👍